The test_ktoblzcheck.py python script does not seem to work with
python3. Nonetheless, python is required to be installed as it
is used to convert the bankdata elsewhere during the build process.

diff -upr ktoblzcheck-1.52.orig/src/python/CMakeLists.txt ktoblzcheck-1.52/src/python/CMakeLists.txt
--- ktoblzcheck-1.52.orig/src/python/CMakeLists.txt	2020-02-04 18:27:25.485349055 +0100
+++ ktoblzcheck-1.52/src/python/CMakeLists.txt	2020-02-04 18:28:11.732029981 +0100
@@ -1,7 +1,7 @@
 
 install(FILES ktoblzcheck.py DESTINATION ${Python_SITEARCH})
 
-if(Python_Interpreter_FOUND)
+if(FALSE)
 add_test(
     NAME test-python
     COMMAND ${Python_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test_ktoblzcheck.py
